uk.et1.big-finish-downloader.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>uk.et1.big-finish-downloader</id>
+  
+  <name>Big Finish Downloader</name>
+  <summary>A GTK app to download your Big Finish Collection</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <icon type="remote">https://raw.githubusercontent.com/Emersont1/big-finish-downloader/main/icons/tardis.svg</icon>
+  <description>
+    <p>
+      This is a program downloads your titles from the big finish website into a folder of your choosing
+    </p>
+  </description>
+  <content_rating type="oars-1.1" />
+  <url type="homepage">https://github.com/Emersont1/big-finish-downloader</url>
+  <releases>
+    <release version="1.0.0.3" date="2021-01-15"/>
+	</releases>
+  
+  <launchable type="desktop-id">Big-Finish-Downloader.desktop</launchable>
```

Comment:
```suggestion
  <launchable type="desktop-id">uk.et1.big-finish-downloader.desktop</launchable>
```
You should rename the desktop file upstream along with upstreaming the metainfo

Reply:
A lot (about half of the most recently committed 10 or so) of repos have local metainfo files in their repo instead of upstream, is that not common practice